tests/nemo_text_processing/es/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
could you change new header like here? Some for other files? https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/inverse_text_normalization/de/taggers/measure.py#L1